### ISSUE TITLE
Request modification that several `lane` can be run in parallel.

### ISSUE THREAD

**Issue by Woollim** [fastlane/fastlane#15841]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
Hello. first of all, thank you very much for developing `fastlane swift`.
`fastlane swift` made automation build system that iOS engineer can understand

But because of the big problem with `fastlane swift`,  we can’t use `fastlane swift` as an automation build system comment line.

`fastlane swift` has a problem that the `lane` of a project in different folders does not work at the same time.

We checked the code, we found that the socket port was fixed at `2000` when Ruby & Swift communicating, and that the other lane can't run when one lane is running.

Due to these problems, `fastlane swift` can't be used as an automation build comment line for build server. so we request modification that several `lane` can be run in parallel.

👍 👍 👍 **USE `swift_server_port` option** 👍 👍 👍 

##### Complete output when running fastlane, including the stack trace and command used
```
INFO [2019-12-30 13:49:28.19]: $ ./fastlane/FastlaneRunner lane betaOTA owner 이병찬 desc hello_nice_to_meet_you app_version 1.2.3 env 개발환경 logMode verbose swiftServerPort 2000 > /dev/null
WARN [2019-12-30 13:49:32.95]: Lane Context:
INFO [2019-12-30 13:49:32.95]: {:LANE_NAME=>"betaOTA"}
ERROR [2019-12-30 13:49:32.95]: fastlane failed to receive a connection from the FastlaneRunner binary after 5 seconds, shutting down
ERROR [2019-12-30 13:49:32.95]: fastlane finished with errors
```

### Environment

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 10.15.2                                                        |
| Ruby                        | 2.6.3                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.21.0 (Apple Git-122.2)                           |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | Mac OS X 10.15.2 (19C57)                                       |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib |
| OpenSSL Version             | LibreSSL 2.8.3                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 11.3                                                           |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | ko_KR.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile.swift`</summary></details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.138.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.3.1        |
| slack-notifier            | 2.3.2        |
| atomos                    | 0.1.3        |
| claide                    | 1.0.3        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.6        |
| xcodeproj                 | 1.14.0       |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| terminal-notifier         | 2.0.0        |
| unicode-display_width     | 1.6.0        |
| terminal-table            | 1.8.0        |
| plist                     | 3.5.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.7.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.7.0        |
| tty-cursor                | 0.7.0        |
| tty-spinner               | 0.9.2        |
| babosa                    | 1.0.3        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| excon                     | 0.71.1       |
| faraday                   | 0.17.1       |
| unf_ext                   | 0.0.7.6      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20190701 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 0.13.1       |
| fastimage                 | 2.1.7        |
| gh_inspector              | 1.1.3        |
| mini_magick               | 4.9.5        |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.3.0        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 1.0.0        |
| dotenv                    | 2.7.5        |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.6        |
| jwt                       | 2.1.0        |
| uber                      | 0.1.0        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| mime-types-data           | 3.2019.1009  |
| mime-types                | 3.3          |
| multi_json                | 1.14.1       |
| signet                    | 0.12.0       |
| memoist                   | 0.16.2       |
| os                        | 1.0.1        |
| googleauth                | 0.6.7        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.23.9       |
| google-cloud-env          | 1.3.0        |
| google-cloud-core         | 1.4.1        |
| digest-crc                | 0.4.1        |
| google-cloud-storage      | 1.16.0       |
| emoji_regex               | 1.0.1        |
| CFPropertyList            | 3.0.1        |
| json                      | 2.2.0        |
| bundler                   | 2.0.2        |
| forwardable               | 1.2.0        |
| logger                    | 1.4.1        |
| stringio                  | 0.0.2        |
| ipaddr                    | 1.2.2        |
| openssl                   | 2.1.2        |
| ostruct                   | 0.1.0        |
| strscan                   | 1.0.3        |
| date                      | 2.0.0        |
| fileutils                 | 1.3.0        |
| io-console                | 0.4.8        |
| zlib                      | 1.0.0        |
| libxml-ruby               | 3.1.0        |
| rexml                     | 3.2.3        |
| psych                     | 3.1.0        |
| mutex_m                   | 0.1.0        |
</details>

*generated on:* **2019-12-30**
</details>

**Comment by Woollim** [comment]
@janpio What's the status quo? I can't use `fastlane swift` until this problem is resolved... 😿

**Comment by janpio** [comment]
Feel free to create a pull request that fixes the problem. This is an Open Source project, so we mostly manage contributions by the community or solve our own problems. As I am not using Swift, this is not one of my problems that I will solve any time soon.

**Comment by Woollim** [comment]
Thank you for your reply. I looked around `fastlane` code to solve this problem, but I was able to solve this problem through the option of ‘swift_server_port’ that already exists!
(Now we can use `fastlane swift` in earnest! Thanks a lot!)

I want to add this experience in `fastlane swift` Docs How about your opinion? @janpio

**Comment by janpio** [comment]
That sounds like a _very_ good idea! https://github.com/fastlane/docs/edit/master/docs/getting-started/ios/fastlane-swift.md is the file to edit here.
